static/src/stylesheets/amp/_hosted.scss

```diff
@@ -0,0 +1,848 @@
+            width: 100%;
+            height: 100%;
+            z-index: 2;
+            background: linear-gradient(to top, rgba(51, 51, 51, .4) 0%, rgba(51, 51, 51, .4) 2%, transparent 100%);
+        }
+    }
+
+    &.hosted-page--bright .content__article-body p.intro + p::first-letter {
+        color: #ffffff;
+    }
+
+    .gs-container {
+        height: 100%;
+    }
+
+    .title {
+        position: absolute;
+        margin: auto;
+        z-index: 3;
+        bottom: 20px;
+        color: #ffffff;
+        font-size: 32px;
+        line-height: 36px;
+        font-weight: bold;
+    }
+
+    .caption {
+        position: absolute;
+        left: 10px;
+        top: 390px;
+        right: unset;
+        padding: 0;
+    }
+
+    figure {
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+    figcaption {
+        font-size: 12px;
+        line-height: 16px;
+        color: $neutral-2;
+        margin-top: 3px;
+    }
+
+    .content__meta-container {
+        margin: #{4 * $gs-baseline} 0 $gs-baseline;
+    }
+
+    .meta__extras {
+        border-top: none;
+    }
+
+    .social__item--pinterest {
+        display: inline-block;
+    }
+
+    .content__article-body p {
+        font-size: 18px;
+        line-height: 24px;
+    }
+
+    .content__article-body p.intro + p::first-letter {
+        display: inline-block;
+        float: left;
+        color: #000000;
+        padding: 10px 16px 12px;
+        font-size: 480%;
+        line-height: 68px;
+        width: 77px;
+        height: 77px;
+        margin-right: 8px;
+        margin-top: 4px;
+        text-align: center;
+    }
+
+    .content__article-body .intro {
+        font-size: 20px;
+        line-height: 24px;
+        font-weight: 100;
+    }
+
+    h1 {
+        line-height: 28px;
+    }
+
+    .hosted__standfirst {
+        padding: 0;
+    }
+
+    .hosted__terms {
+        width: 100%;
+        margin-top: 0;
+        padding: 10px 0;
+        border-top: 2px solid #eeeeee;
+    }
+
+    .from-content-api ul > li:before {
+        border-radius: 0;
+        width: 8px;
+        height: 8px;
+        margin: 1px 8px;
+    }
+
+    .from-content-api > ul li {
+        margin-bottom: 0;
+    }
+
+    .gu-image {
+        height: auto;
+    }
+}
+
+.hosted__next-page {
+    margin: $gs-baseline 0;
+    position: relative;
+}
+
+.hosted__next-page--header {
+    margin: 0;
+    border: 0;
+    padding: 0;
+}
+
+.hosted__next-page-header--border {
+    width: 100%;
+    height: 10px;
+    margin-bottom: 4px;
+}
+
+.hosted__next-page-title {
+    font-size: 14px;
+    line-height: 16px;
+    padding: 6px;
+    min-height: 42px;
+}
+
+.hosted-video-page {
+    .hosted__next-page {
+        margin: $gs-baseline 0 0;
+        padding: 0 $gs-gutter $gs-baseline;
+        background: $neutral-7;
+    }
+
+    .hosted__next-page-header--border {
+        margin-left: -$gs-gutter;
+        width: 150%;
+    }
+
+}
+
+.hosted-gallery-page__main-body {
```

Comment:
Would it make sense to have hosted galleries have its own stylesheet? This one is massive and seems like it is touching a lot of different types of things. Might be worth breaking it down more.

You could event conditionally only load it on things that are hosted and galleries.

On that note though, galleries aren't currently supported in AMP, how are you doing them and what do they look like?